zebra-network/src/config.rs

```diff
@@ -176,3 +184,86 @@ impl Default for Config {
         }
     }
 }
+
+impl<'de> Deserialize<'de> for Config {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        #[derive(Deserialize)]
+        #[serde(deny_unknown_fields, default)]
+        struct DConfig {
+            listen_addr: String,
+            network: Network,
+            initial_mainnet_peers: HashSet<String>,
+            initial_testnet_peers: HashSet<String>,
+            peerset_initial_target_size: usize,
+            #[serde(alias = "new_peer_interval")]
```

Comment:
Do we need to delete the alias in `Config`?

Reply:
I do not know. It depends. If we do not need to support the alias for `crawl_new_peer_interval` we can remove it.

Reply:
We do need to support the alias, so it can stay.